src/assertions/focus.js

```diff
@@ -0,0 +1,21 @@
+import elementExists from '../util/element-exists';
+
+export default function focus(client, chai, utils) {
+    chai.Assertion.addMethod('focus', function() {
+        const selector =  utils.flag(this, 'object');
+        const negate = utils.flag(this, 'negate');
+        const immediately = utils.flag(this, 'immediately');
+
+        if (!immediately) {
+            elementExists(client, selector, negate);
```

Comment:
I see, I think I was misunderstanding how 'negate' was using inside of element exists. 

Basically, we don't want to wait for an element not to exist, just to confirm that the element doesn't have focus (b/c it obviously won't). 

I'll remove the 'negate' part. 👍